base/container_scripts/run_functional_tests.sh

```diff
@@ -43,7 +38,12 @@ EOF
 
 source "/opt/oci_env/base/container_scripts/configure_pulp_smash.sh"
 
-cd "/src/$PACKAGE/"
+if [ $PACKAGE == "pulp_file" ]
+then
+    cd "/src/pulpcore/${PACKAGE}"
```

Comment:
```suggestion
    cd "/src/pulpcore/"
```
The pytest command is `$PROJECT.tests.functional` so I think we only need to cd into the top level of pulpcore.